src/MountedTraversal.js

```diff
@@ -177,6 +178,12 @@ export function parentsOfInst(inst, root) {
   return pathToNode(inst, root).reverse();
 }
 
+export function instMatchesObjectProps(inst, props) {
+  if (!isDOMComponent(inst)) return false;
+  const node = getNode(inst);
+  return !isEmpty(props) && isMatch(propsOfNode(node), props);
```

Comment:
> I agree this doesn't seem like a normal use case, although I can imagine some code that builds up some object dynamically, and in some cases might build up to nothing...

Exactly. But I would say that's a bug in their code. If they are dynamically building an object, and it gets to an empty state. An error should be thrown, so they can make sure they are testing what they want.

> I don't feel strongly about this, but I don't see the harm in having {} match all nodes. It seems like the expected behavior, despite being a "corner-case" type input?

Would you also want to see `find('[]')` to match every node? Because that is the same API effectively. 

Comparing it to the `'[]'` api, I would think it should only find components with props that match the specified key/values.